modules/accessing-microshift.adoc

```diff
@@ -0,0 +1,46 @@
+= Accessing the MicroShift cluster
+== Installing the OpenShift client
+
+You can install the OpenShift CLI (oc) client on your RHEL machine by running:
+
+[source,terminal]
+----
+sudo dnf install -y openshift-clients
+----
+
+== Accessing the MicroShift cluster locally
+
+To access the MicroShift cluster from the _same_ machine that is running MicroShift, copy the generated `kubeconfig` file to your user's `~/.kube/config` file:
+
+[source,terminal]
+----
+mkdir -p ~/.kube/
+sudo cat /var/lib/microshift/resources/kubeadmin/kubeconfig > ~/.kube/config
+----
+
+== Accessing the MicroShift cluster remotely
+
+To access the MicroShift cluster from a _different_ machine from the one running MicroShift, use the following procedure:
+
+1. Open the Kubernetes API port in the firewall of your MicroShift machine:
+
+    MICROSHIFT_MACHINE=<name or IP of your MicroShift machine>
+    ssh $MICROSHIF_MACHINE "sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp && sudo firewall-cmd --reload"
```

Comment:
```suggestion
    ssh $MICROSHIFT_MACHINE "sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp && sudo firewall-cmd --reload"
```

Reply:
In addition, this implies a running firewalld. Should we use `firewall-offline-cmd`?

Reply:
`firewalld` is included and started by default on RHEL systems (except for RHEL cloud images that rely on the cloud provider's FW). `firewall-offline-cmd` can only be used if `firewalld` isn't running.